tests/common/util/http_helpers_test.py

```diff
@@ -47,22 +49,24 @@ class HttpHelpersTest(ForsetiTestCase):
     """Test http_helpers utility functions."""
 
     def test_without_suffix(self):
+        http_helpers._PLATFORM = "gke"
```

Comment:
I think we should leave existing unit tests as is and add new ones for the platform changes. Instead of `http_helpers._PLATFORM = "gke"` you can set the env variable in the test setup like this `os.environ['FORSETI_PLATFORM'] = 'gke'`.

Reply:
@gkowalski-google Please remind me, if I set this in the test setup, won't it be set for each unit test?

Reply:
Yes, sorry for the confusing words. I just meant in the test setup/arrangement inside of the unit test you want to test this. I'd say add one test and verify platform is gce in the UA, and then one test where you set the env var to 'gke' and then verify that.

Reply:
Got it.  Will do.

Reply:
I remember why I set the attribute.  If I set `os.environ['FORSETI_PLATFORM'] = 'gke'` in the test, the http_helpers won't honor it.

Reply:
But, I think I can mock the environment variable.